infrastructure_playbook_repo.md

```diff
@@ -0,0 +1,179 @@
+| [usegalaxy-eu.gie-node-proxy](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/usegalaxy-eu.gie-node-proxy) |  |  | Clones [GIE NodeJS proxy](https://github.com/galaxyproject/gx-it-proxy) configurations and installs Node dependencies and sets up the [GIE proxy](https://docs.galaxyproject.org/en/release_20.09/admin/special_topics/interactive_environments.html#setting-up-the-proxy)  |
+| [usegalaxy-eu.google-verification](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/usegalaxy-eu.google-verification) |  |  | Adds Google site verification HTML file and adds required Nginx configuration |
+| [usegalaxy-eu.grt-client](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/usegalaxy-eu.grt-client) |  |  | Adds cron jobs that can export and upload data to [GRT](https://github.com/hexylena/galactic-radio-telescope) |
+| [usegalaxy-eu.grt-export](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/usegalaxy-eu.grt-export) |  |  | Adds a cron job that exports data to GRT |
+| [usegalaxy-eu.htcondor_release](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/usegalaxy-eu.htcondor_release) |  | :heavy_check_mark: | Adds a cron job that releases Condor jobs that are in hold state _(also removes jobs in hold state that are resubmitted more than two times)_ |
+| [usegalaxy-eu.jenkins-ssh-key](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/usegalaxy-eu.jenkins-ssh-key) |  | :heavy_check_mark: | Creates SSH directory and adds a key to the Jenkins user |
+| [usegalaxy-eu.log-cleaner](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/usegalaxy-eu.log-cleaner) |  | :heavy_check_mark: | Adds cron job to clean up old journalctl logs of gunicorn and galaxy handlers services |
+| [usegalaxy-eu.logrotate](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/usegalaxy-eu.logrotate) |  | :heavy_check_mark: | Adds logrotate configuration for galaxy and atop logs  |
+| [usegalaxy-eu.monitoring](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/usegalaxy-eu.monitoring) |  | :heavy_check_mark: | Adds Telegraf tasks for monitoring NFS shares access times, and to collect NFS stats |
+| [usegalaxy-eu.plausible](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/usegalaxy-eu.plausible) |  | :heavy_check_mark: | Clones [Plausible](https://github.com/plausible/analytics)  Analytics [setup](https://github.com/usegalaxy-eu/hosting) and adds the configuration and starts the service using Docker |
+| [usegalaxy-eu.remap-user](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/roles/usegalaxy-eu.remap-user) |  |  | Remaps system user and group with UID and GID 999 to a different UID and GID so Galaxy user can be created with 999 UID and GID |
```

Comment:
Isn't this obsolete and replaced by the Handy_OS collection/role?

Reply:
Yes, same as my previous comment.